reference/3.0/Microsoft.PowerShell.Core/About/about_Ref.md

```diff
@@ -5,72 +5,156 @@ locale:  en-us
 keywords:  powershell,cmdlet
 title:  about_Ref
 ---
-# About Ref
-
-## SHORT DESCRIPTION
 
-Describes how to create and use a reference variable type.
+# About Ref
 
-## LONG DESCRIPTION
+## Short description
 
-You can use the reference variable type to permit a method to change the value
+Describes how to create and use a reference type variable. You can use
+reference type variables to permit a method to change the value
 of a variable that is passed to it.
 
-When the [ref] type is associated with an object, it returns a reference to
-that object. If the reference is used with a method, the method can refer to
-the object that was passed to it. If the object is changed within the method,
-the change appears as a change in the value of the variable when control
-returns to the calling method.
+## Long description
 
-To use referencing, the parameter must be a reference variable. If it is not,
-an InvalidArgument exception is thrown.
+You can pass variables to methods *by reference* or *by value*.
 
-The parameters used in method invocations must match the type required by the
-methods.
+In the following example, a method attempts to change the value of a
+variable that is passed into it, and fails.
 
-Examples:
+When you pass a variable *by value*, you are passing a copy of the data.
```

Comment:
Rearrange this as follows:

When you pass a variable *by value*, you are passing a copy of the data.

In the following example, the function changes the value of the variable passed to it. In PowerShell, integers are value types so they are passed *by value*. Therefore, the value of `$var` is unchanged outside the scope of the function.

Reply:
check